common/middleware/process-original-request-body.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function processOriginalRequestBody(authMountpoint = '/auth') {
```

Comment:
It looks like this is missing unit tests. It would be good to add some for this as there is a bit of logic.

Reply:
My local `.gitignore` was hiding it :(